platform-controller/src/main/java/org/hobbit/controller/orchestration/ContainerManager.java

```diff
@@ -0,0 +1,84 @@
+package org.hobbit.controller.orchestration;
+
+import org.hobbit.controller.docker.ContainerStateObserver;
+
+import java.util.List;
+
+public interface ContainerManager<ReplicationController, Metrics> {
```

Comment:
Again a class that has been moved and that "lost" a lot of its documentation. Please add the documentation back. It is really important to have the methods documented.

Apart from that, I am wondering why you have added `<ReplicationController, Metrics>` to the class. What is the rational behind that? This would only make sense if for example the two metrics classes would share a common super type. As far as I know this is not the case.